src/ext/java/org/opentripplanner/ext/vectortiles/layers/stops/DigitransitStopPropertyMapper.java

```diff
@@ -46,12 +48,9 @@ public Collection<T2<String, Object>> map(TransitStopVertex input) {
       patternsForStop
         .stream()
         .map(tripPattern -> {
-          int stopPos = tripPattern.findStopPosition(stop);
-          var headsign = stopPos < 0
-            ? "Not Available"
-            : tripPattern.getScheduledTimetable().getTripTimes().get(0).getHeadsign(stopPos);
+          String headsign = tripPattern.getStopHeadsign(tripPattern.findStopPosition(stop));
           JSONObject pattern = new JSONObject();
-          pattern.put("headsign", headsign);
+          pattern.put("headsign", Optional.ofNullable(headsign).orElse("Not Available"));
```

Comment:
This doesn't change it but I'm not sure if I like this "Not available" default value. Can't it just be an empty string?

Reply:
Fixed